docs/src/utils/Ph/index.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import tinycolor from 'tinycolor2';
+
+import { phOuter, phInner, phText } from './styles.module.css';
+
+function Ph({ children, bg, h, aspect }) {
```

Comment:
what does Ph stand for? can we have a minimal docstring?

Reply:
Answered above 😬 😃